pyramda/prop.py

```diff
@@ -0,0 +1,8 @@
+from pyramda.function.curry import curry
+
+
+@curry
+def prop(x, obj):
+    if isinstance(obj, dict):
```

Comment:
Huh, I'd completely forgotten that `getitem` was already implemented in the first place. Oh well :)